### ISSUE TITLE
Naming consistency "Lisk"

### ISSUE THREAD

**Issue by kuy** [kirillshevch/react-cryptocoins#9]
Hello, I found an inconsistent name: `Lisk`. Lisk's symbol is `LSK`.

```
export LiskAlt from './icons/LISK-alt';
export Lisk from './icons/LISK';
```

Can I send PR to fix this?

**Comment by kirillshevch** [comment]
Ok, but you need to solve this through condition in [build.js](https://github.com/kirillshevch/react-cryptocoins/blob/master/bin/build.js) for keep naming for the next versions

**Comment by kuy** [comment]
Umm, I didn't notice that symbol name is coming from SVG files of `allienworks/cryptocoins` repository.

**Comment by kirillshevch** [comment]
Solved in #12
